### PR TITLE
typo fix in traefik/README.md

### PR DESCRIPTION
**Pull request by AdamDang**
Fix some capitalizing typos.
In  a same table, the same word uses different format.

### CODE REVIEW

File: stable/traefik/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: traefik
-version: 1.28.1
+version: 1.28.2
```
Comment (krancour) [inline stable/traefik/Chart.yaml:3]: Needs to be 1.29.1 (or 1.29.2 if #4608 is merged first).

**Comment by rjkernick** [comment]
Please fix the merge conflict

**Comment by AdamDang** [comment]
@rjkernick
According to my experience, it should be recreated with such conflicts with changes caused by other PR. I will recreate now.
Thank you!

**Comment by AdamDang** [comment]
@rjkernick 
The new PR is #5134
Thanks!

**Comment by AdamDang** [comment]
closing

**Comment by krancour** [comment]
@AdamDang  for future reference, you do not need to close PRs and re-open them when this happens. You do this:

```
$ git checkout master
$ git pull <upstream remote> master
$ git checkout <feature/fix branch>
$ git rebase master
## ... Fix merge conflicts ...
$ git rebase --continue
$ git push <remote for your fork-- usually origin> <feature/fix branch> -f
```

That last line is a force push. It will automatically update your PR.

**Pull request closed without merging**